docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -1134,6 +1140,9 @@ These parameters are associated with the fixed (i.e., static) files. On `Level 1
 ``FIXlut``: (Default: "")
    System directory where the lookup tables for optics properties are located.
 
+``FIXshp``: (Default: "")
+   System directory where the graphics shapefiles are located. On Level 1 systems, these are set within the machine files. Users on other systems will need to provide the path to the directory that contains the *Natural Earth* shapfiles.
```

Comment:
Should *Natural Earth* shapfiles be shapefiles?

Reply:
Good catch! Apparently my brain was on vacation leading up to Christmas. ;)